receiver.py

```diff
@@ -0,0 +1,27 @@
+from datetime import datetime, timedelta
+import csv
+
+file= open('Ambewela.txt.','r')
```

Comment:
I can see this Ambewela.txt and waga.csv in the code. Please assign them to proper variables and use as so.

Reply:
check now..